packages/customize-widgets/src/components/inserter/style.scss

```diff
@@ -0,0 +1,60 @@
+$customize-widgets-header-width: 50px;
```

Comment:
Should this be `height`, not `width`?

Reply:
I think we should try and match the height of the customizers action area, which is 46px:

<img width="315" alt="image" src="https://user-images.githubusercontent.com/191598/111187521-729ffa00-858a-11eb-8c52-af306ce2e3ac.png">